README.md

```diff
@@ -12,15 +12,18 @@ npm install react-inline-confirm --save
 
 ## How to Use
 
+```isExecuting``` is the state of react when the button was clicked. The default value should be false. It can be passed as an argument for your react component.
```

Comment:
haha, I did read. I was just updating it with your comment without the optional part.